l10n_br_simple/data/account_group_data.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="account_group_1" model="account.group">
```

Comment:
Can you rename all the records with a mor comprehensive name?
For example ACCOUNT_GROUP_ATIVO

Reply:
@mileo,

In this case we use in XML ref the code account, there is no reason to use account name because if you want load a custom account you cant reuse this account group in reports